{{cookiecutter.repo_name}}/LICENSE

```diff
@@ -1,101 +1,674 @@
-{% if cookiecutter.open_source_license == 'MIT' %}
-MIT License
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
```

Comment:
Is everyone happy with GPL v3? Or should it be GPL v2+ to match the main repo?

Reply:
Go 2+ also yes this is the only option